playbooks/ansible-cloud/vsca-appliance/files/bootstrap.yaml

```diff
@@ -23,6 +23,35 @@
         __password: "{{ lookup('password', '/dev/null length=15 chars=ascii_letters') }}"
       no_log: true
 
-    - name: Change zuul user password
-      shell: "echo -e 'zuul\n{{ __password }}\n{{ __password }}' | passwd zuul"
-      no_log: true
+    - name: Update zuul password
+      user:
+        name: "zuul"
+        password: "{{ __password | password_hash('sha512') }}"
+      become: true
+
+    - name: Write the /etc/hosts file
+      copy:
+        content: |
+          127.0.0.1  localhost
+          ::1  localhost ipv6-localhost ipv6-loopback
+          {{ hostvars['vmware-vcsa-6.7.0']['nodepool']['private_ipv4']}} vcenter.test vcenter
+        dest: /etc/hosts
+      become: true
+      register: hosts_status
+
+    - name: Unconditionally reboot the machine with all defaults
```

Comment:
I suspect this is going to be really expensive to do, we any info on what you are trying to do?

Reply:
It's a hack to restart all the services with the correct reverse DNS. The reboot takes 30s.
This will go away the day I find a better solution.